experimental/adaptive-cards-2/README.md

```diff
@@ -0,0 +1,42 @@
+# Adaptive Cards Schema 2.0 Sample
+
+This bot has been created using [Bot Framework](https://dev.botframework.com). It shows how to create a simple bot that returns a series of Adaptive Cards that leverage the Adaptive Card 2.0 schema that includes Action.Execute and the Invoke Activity processing to handle these Action.Execute requests.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## Key concepts in this sample
+
+The solution includes a Catering bot that asks users for an entre and drink order. Orders are stored in a Cosmos DB. The key conecepts include:
+
+1. How to return an Adaptive Card that uses schema 2.0
+2. How to use Action.Execute in the JSON of an Adaptive Card
+3. How to perform an automatic card refresh of an Adaptive Card 2.0
+4. How to process the Invoke Activities that an Action.Execute emits
+5. How to respond to an Action.Execute Invoke
+6. How to update Web Chat to use Adaptive Cards 2.0
+
+## To try this sample
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+- Create a bot registration in the azure portal for the `Catering Bot` and update [appsettings.json](CateringBot)
+- Add the Microsoft AppID and Password to the appsettings.json file
```

Comment:
Does a developer need to replace the AppId values in each AdaptiveCard? It might be better to have the developer perform a grep  and replace in this scenario.

Reply:
It is an artifact of the current Adaptive Cards preview library that is included. The AppID property is required by the AC library, but it doesn't use the value. This is/will change.

Reply:
So it can stay all those '1111...' values I put in there.